toolbar/core/src/components/toolbar/toolbar.tsx

```diff
@@ -255,7 +154,7 @@ export function ToolbarDraggableBox() {
           )}
         >
           {/* Render content based on state */}
-          {isLoadingState && <LoadingStateContent />}
+          {true && <ConnectingStatePanel />}
```

Comment:
Just found this afterwards @glenntws. We need to be more cautious and test more intensely